### PR TITLE
Updated IPv8 pointer

### PR DESCRIPTION
**Pull request by qstokkink**
Related to #5450

This PR:
 - Updates the IPv8 pointer, resolving #5450.
 - Explicitly sets `working_directory` of the IPv8 configuration.

### CODE REVIEW

File: src/tribler-core/tribler_core/session.py [hunk 1]
```diff
@@ -424,6 +424,7 @@ async def start(self):
             ipv8_config['address'] = self.config.get_ipv8_address()
             ipv8_config['overlays'] = []
             ipv8_config['keys'] = []  # We load the keys ourselves
+            ipv8_config['working_directory'] = str(self.config.get_state_dir() / 'sqlite')
```
Comment (devos50) [inline src/tribler-core/tribler_core/session.py:427]: Isn't the `sqlite` directory appended by the IPv8 database logic?

Comment (qstokkink) [reply src/tribler-core/tribler_core/session.py:427]: Good point, is the `self.config.get_state_dir()` safe to write to for IPv8-based overlays or should this go into `self.config.get_state_dir() / 'ipv8'`?

Comment (devos50) [reply src/tribler-core/tribler_core/session.py:427]: `self.config.get_state_dir()` should be safe to write to. Please test it with Tribler to see if it ends up in the right directory (e.g., by reverting the lazy loading commit).

Comment (qstokkink) [reply src/tribler-core/tribler_core/session.py]: @devos50 I made that change and verified:

 - No `identity.db` or `attestations.db` is created on a normal run.
 - When lazy loaded in, `identity.db` and `attestations.db` are created in `.Tribler/7.5/sqlite`

Comment (qstokkink) [reply src/tribler-core/tribler_core/session.py]: Assuming that is expected behavior, I'll mark this as READY.

**Pull request merged**